interpreter/value_test.go

```diff
@@ -3462,19 +3462,26 @@ func TestPublicKeyValue(t *testing.T) {
 			UInt8Value(sema.SignatureAlgorithmECDSA_secp256k1.RawValue()),
 		)
 
-		assert.PanicsWithValue(t,
-			InvalidPublicKeyError{PublicKey: publicKey, Err: fakeError},
-			func() {
-				_ = NewPublicKeyValue(
-					inter,
-					EmptyLocationRange,
-					publicKey,
-					sigAlgo,
-					func(context PublicKeyValidationContext, locationRange LocationRange, publicKey *CompositeValue) error {
-						return fakeError
-					},
+		func() {
```

Comment:
Just curious: Can `assert.PanicsWithValue` not be used, because it compares by value, and the expected / actual pointers are considered different?

Reply:
yeah, exactly.